EnoughCoffee/src/EnoughCoffee.java

```diff
@@ -0,0 +1,44 @@
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.Collections;
+
+public class EnoughCoffee {
+    public static void main(String[] args) throws Exception {
+        BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
```

Comment:
@AraChilingaryan try to split on small methods

Reply:
done